### PR TITLE
[TE] entity anomaly logging for ad-hoc debugging

### PR DESCRIPTION
**Pull request by akshayrai**

### CODE REVIEW

**Review by harleyjj — APPROVED** [review]
Awesome!

File: thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/tools/RunAdhocDatabaseQueriesTool.java [hunk 1]
```diff
@@ -723,9 +741,65 @@ private void rollbackMigrateSubscriptionWatermarks() {
...
+          k++;
+        }
+        j++;
+      }
+      i++;
+    }
+  }
+
+  public static void main(String[] args) throws Exception {
+    File persistenceFile = new File("/Users/akrai/persistence-local.yml");
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/tools/RunAdhocDatabaseQueriesTool.java:802]: hide the user name here?

**Review by jihaozh — APPROVED** [review]
LGTM

**Review by vincentchenjl — APPROVED** [review]
LGTM!

**Pull request merged**